### ISSUE TITLE
O que transmitir

### ISSUE THREAD

**Issue by eduzal** [fazolin/KinectBand#1]
não seria mais interessante transmitir 2 arrays ou arraylists de Vec3, um de coordenadas e outro de cores RGB? ao invés de enviar um webp pra procesar server side?

**Comment by fazolin** [comment]
Sim, e podemos começar transmitindo somente depth, que já é muito bom. Podemos transmitir somente o Z, pois XY já é dado.

**Comment by fazolin** [comment]
Transferi a referência para a wiki, aí colocamos tudo nesse doc
https://github.com/fazolin/KinectBand/wiki/Kinect-Band---Estrutura-B%C3%A1sica
